src/app/components/admin-components/admin/admin.component.ts

```diff
@@ -0,0 +1,42 @@
+import {Component, OnInit} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+import {CategoryStatsService} from '@app/_services/api/admin/category-stats.service';
+import {QuestionCountService} from '@app/_services/api/admin/question-count.service';
+import {QuestionCount} from '@app/_models/question_counts';
+import {CategoryStats} from '@app/_models/category_stats';
+import {UserStatsService} from '@app/_services/api/user-stats.service';
+import {UserStats} from '@app/_models';
+
+@Component({
+    selector: 'app-admin',
+    templateUrl: './admin.component.html',
+    styleUrls: ['./admin.component.scss']
+})
+export class AdminComponent implements OnInit {
+    questionAPIData: QuestionCount[];
```

Comment:
questionAPIData is a bad name. questionData or questionCount would be better.

Reply:
Same for other members.

Reply:
Renamed in admin component